packages/components/typedoc-internal.js

```diff
@@ -4,17 +4,19 @@
  * Licensed under the Apache License, Version 2.0: http://www.apache.org/licenses/LICENSE-2.0
  */
 module.exports = {
-    out: './docs',
+    name: "@labkey/components (internal)",
```

Comment:
Where does the name show up?  (Just curious)

Reply:
<img width="1110" alt="Screen Shot 2020-05-08 at 11 10 10 AM" src="https://user-images.githubusercontent.com/6411206/81425086-871a9700-911c-11ea-93e0-9dc2841c98a7.png">